content/news/new-project-logo.md

```diff
@@ -0,0 +1,43 @@
+Today we're pleased to show off a project that we have been working on for some time: a new logo for OpenRA!
```

Comment:
On my system this line wraps so that only the last word is on the second line. Can you shorten it just a bit, maybe?